endpoints/base

```diff
@@ -146,7 +145,7 @@ function exit_error() {
 function process_opts() {
     local longopts="validate,endpoint-opts:,endpoint-label:,run-id:,base-run-dir:,image:"
     local longopts+=",roadblock-server:,roadblock-passwd:,roadblock-id:,osruntime:,max-sample-failures:"
-    local longopts+=",max-rb-attempts:,endpoint-deploy-timeout:,engine-script-start-timeout:"
+    local longopts+=",,endpoint-deploy-timeout:,engine-script-start-timeout:"
```

Comment:
did you really mean to leave two commas here?

Reply:
No, good catch.